.github/CODEOWNERS

```diff
@@ -0,0 +1,24 @@
+# default group
+* @pcdshub/epics-reviewers
+
+# language-specific group(s)
```

Comment:
maybe this just needs python-reviewers added

maybe it doesn't really matter since this is a fork